libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -172,10 +200,14 @@ const INVOKE_RESPONSE_KEY: symbol = Symbol('invokeResponse');
  * ```
  */
 export class BotFrameworkAdapter extends BotAdapter implements IUserTokenProvider {
-    protected readonly credentials: MicrosoftAppCredentials;
-    protected readonly credentialsProvider: SimpleCredentialProvider;
-    protected readonly settings: BotFrameworkAdapterSettings;
-    private isEmulatingOAuthCards: boolean;
+    private readonly credentials: MicrosoftAppCredentials;
+    private readonly credentialsProvider: SimpleCredentialProvider;
+    private readonly settings: BotFrameworkAdapterSettings;
```

Comment:
Why is this suddenly `private`?

Reply:
We don't know, so they are protected once more!